draft-irtf-cfrg-ristretto.xml

```diff
@@ -39,14 +38,14 @@ a prime-order group.</t>
 <t>Ristretto is a technique for constructing prime-order groups with
 non-malleable encodings from non-prime-order elliptic curves.  It
 extends the <xref target="Decaf"></xref> approach to cofactor elimination to support
-cofactor-8 curves such as Curve25519 <xref target="RFC7748"></xref>. In particular, this
-allows an existing Curve25519 library to provide a prime-order group
+cofactor-8 curves such as Curve25519 <xref target="RFC7748"></xref>. In particular,
+this enables a prime-order group using an existing Curve25519 library
 with only a thin abstraction layer.</t>
 <t>Edwards curves provide a number of implementation benefits for
 cryptography, such as complete addition formulas with no exceptional
 points and the fastest known formulas for curve operations.  However,
-every Edwards curve has a point of order 4, so that the group of
-points on the curve is not of prime order but has a small cofactor.</t>
+every Edwards curve has a point of order 4. Thus, the group of
+points on the curve is not of prime order and has a small cofactor.</t>
```

Comment:
"but" contrasts "prime order" with "small cofactor" in a way that "and" doesn't, but this sentence could indeed be broken up into two parts.